### ISSUE TITLE
[Anon] Bug report: Tyranids - Genestealer Cults.catz

### ISSUE THREAD

**Issue by BSDataAnon** [BSData/wh40k#1843]
**File:** Tyranids - Genestealer Cults.catz

**BattleScribe version:** 2.01.07

**Platform:** iPhone / iPod / iPad

**Dropbox:** No

**Description:** Neophyte acolyte heavy weapons team is point d out at 5pts and it should be 10. 5 per each of the two neophytes that form a heavy weapon team.

**Comment by GenWilhelm** [comment]
A Neophyte Hybrid unit costs 5 points per model in the unit. A weapons team is a single model, ergo it increases the cost of the unit by 5 points.

**Comment by bbermanbates** [comment]
Unit entry specifically says a “two Neophyte Hybrids May form a single Neophyte Heavy Weapons team.” The cost entry for the unit only lists the Neophyte hybrid model with a point cost of 5/each. This a Heavy Weapons Team is purchased with two 5 point Neophyte Hybrids, 10pts.”

**Comment by alphalas** [comment]
@genwilhelm he’s right; that’s how all the Guard stuff is too.

**Comment by GenWilhelm** [comment]
The points section doesn't list the cost of any specific models, only the cost of units. As I said earlier, the Neophyte Hybrids unit costs 5 points per model.

Unless it says somewhere that a Neophyte Weapons Team counts as two models, I see no reason why it should increase the cost of the unit by anything other than 5pts.

**Comment by bbermanbates** [comment]
It literally says that 2 Neopytes form a Heavy Weapons squad.

**Comment by GenWilhelm** [comment]
Yes, so the two models from the default squad are no longer part of the unit, and therefore do not have to be paid for.

**Comment by GenWilhelm** [comment]
After some lengthy internal discussion, we've come to an impasse. The only conclusion drawn is that one of the following is the case:

1. Each weapons team is two models, and therefore must be paid for and played as such.
2. Each weapons team is one model, and therefore must be paid for and played as such.
3. Each weapons team is one model, but should be errata'd to represent the cost of two other models.

Hopefully the upcoming Codex Approved, and subsequent FAQs, can put the issue to bed.

**Comment by amis92** [comment]
@GenWilhelm I think there was some recent discussion on Gitter, but can't remember what the outcome was. We're past Big FAQ. Let's close this issue, either way. I'd hope to have both AM and GSC versions behave consistently, but it's your call.
